cmd/run.go

```diff
@@ -54,7 +54,7 @@ func newRunCmd(runFunc func(context.Context, app.Config) error) *cobra.Command {
 }
 
 func bindRunFlags(flags *pflag.FlagSet, config *app.Config) {
-	flags.StringVar(&config.LockFile, "lock-file", ".charon/cluster-lock.json", "The path to the cluster lock file defining distributed validator cluster")
+	flags.StringVar(&config.LockFile, "lock-file", ".charon/cluster/cluster-lock.json", "The path to the cluster lock file defining distributed validator cluster")
```

Comment:
not sure about this. What is the reasoning behind the change?

Reply:
sorry, unwanted change (in my head, i was thinking of charon create cluster which puts cluster-lock.json inside .charon/cluster/cluster-lock.json)